app/controllers/Controller.scala

```diff
@@ -67,7 +68,8 @@ abstract class LoggedInController extends BaseController with AuthElement {
   }
 }
 
-case class ApplicationRequest[A](application: Application, role: Role, user: DeveloperSession, request: Request[A]) extends WrappedRequest[A](request)
+case class ApplicationRequest[A](application: Application, subscriptions: Seq[APISubscriptionStatus], role: Role, user: DeveloperSession, request: Request[A])
```

Comment:
@AndySpaven : This is the 'headline change' where we moved the subs into the uplifted app request (and no longer has a 'view model' object.